src/main/java/mate/academy/springboot/datajpa/service/mapper/CategoryMapper.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.springboot.datajpa.service.mapper;
+
+import mate.academy.springboot.datajpa.dto.request.CategoryRequestDto;
+import mate.academy.springboot.datajpa.dto.response.CategoryResponseDto;
+import mate.academy.springboot.datajpa.model.Category;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CategoryMapper implements ResponseDtoMapper<CategoryResponseDto, Category>,
+        RequestDtoMapper<CategoryRequestDto, Category> {
+    @Override
+    public Category mapToModel(CategoryRequestDto dto) {
+        Category category = new Category();
+        category.setName(dto.getName());
+        return category;
+    }
+
+    @Override
+    public CategoryResponseDto mapToDto(Category category) {
+        CategoryResponseDto categoryResponseDto = new CategoryResponseDto();
+        categoryResponseDto.setName(category.getName());
+        categoryResponseDto.setId(category.getId());
+        return categoryResponseDto;
+    }
```

Comment:
```suggestion
    @Override
    public CategoryResponseDto mapToDto(Category model) {
        CategoryResponseDto dto = new CategoryResponseDto();
        dto.setName(model.getName());
        dto.setId(model.getId());
        return dto;
    }
```
same everywhere in mappers